TimeFormat.java

```diff
@@ -9,9 +9,40 @@ public static void main(String[] args) {
 		// It concatenates the empty string "" with the leftmost hour-digit. 
 		// It then concatenates the resulting string with the rightmost hour-digit,
 		// and then uses parseInt to cast the resulting string as an int.
+
+		
+		
 		int hours = Integer.parseInt("" + args[0].charAt(0) + args[0].charAt(1));
 		// Does the same with the minutes part of the input.
 		int minutes = Integer.parseInt("" + args[0].charAt(3) + args[0].charAt(4));
         // Replace this comment with the rest of your code
+		//String realTime = [1]
+		//if (time.length() == 3)
+		//{
+			
+		//}
+		String timeFormat = "AM";

```

Comment:
it is not expected.. and is more high level, but I think it is a bad name.. 
When people discuss timeFormat the meaning is "h:mm" or "hh:mm" or "hh:mm AA" etc. not "AM"/"PM", those are include timeFormat include in it. 
There is no expectation for you to know this and this is not covered in the course, but you should avoid using the class name if you can.. since it is TimeFormat.java
This is not ideal.. amPmString or dayTime would be better, 
or if you want a boolean you can use isAM or isPM etc.